boards/sky-drones/smartap-airlink/init/rc.board_sensors

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+#
+# SMARTAP AIRLINK specific board sensors init
+#------------------------------------------------------------------------------
+board_adc start
+
+# Internal SPI bus MPU-9250 / SPI 2 [FPC]
+mpu9250 -s -b 2 -M -R 6 start
+
+# Internal SPI bus ICM-20602 / SPI 1
+icm20602 -R 0 -s start
+
+# Internal SPI bus MPU-9250 / SPI 3
+mpu9250 -s -b 3 -M -R 4 start
+
+# Internal compass
+bmm150 -I start
+
+# Internal Baro
+ms5611 -X start
+
+# Internal Baro
+bmp388 -I -a 0x77 start
+#bmp388 -I start
+
+# External HMC5983
+hmc5883 -T -X -R 4 start
```

Comment:
Is this a specific sensor that is normally present and expected to have that rotation?

Reply:
This is a magnetometer on externally connected GPS module which is normally used. The rotation is related to the sensor on that GPS module.